docs/integrations/databases/supabase.mdx

```diff
@@ -3,170 +3,672 @@ title: Integrate Supabase with Clerk
+    - Under **Signing key**, add the value of your Supabase **JWT secret key** from [the previous step](#get-your-supabase-jwt-secret-key). 
+    - Leave all other fields at their default settings unless you want to customize them. See [Clerk's JWT template docs](/docs/backend-requests/making/jwt-templates#creating-a-template) to learn what each of them do.
+    - Select **Apply changes** to complete setup.
+
+### Set up your local project
+
+To use Clerk with Supabase in your code, first install the necessary SDKs by running the following terminal command in the root directory of your project:
+
+<Tabs type="framework" items={["Next.js", "React"]}>
+  <Tab>
+    <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+      ```bash filename="terminal"
+      npm install @clerk/nextjs @supabase/supabase-js
+      ```
+
+      ```bash filename="terminal"
+      yarn add @clerk/nextjs @supabase/supabase-js
+      ```
+
+      ```bash filename="terminal"
+      pnpm add @clerk/nextjs @supabase/supabase-js
+      ```
+    </CodeBlockTabs>
+  </Tab>
+  <Tab>
+    <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+      ```bash filename="terminal"
+      npm install @clerk/clerk-react @supabase/supabase-js
+      ```
+
+      ```bash filename="terminal"
+      yarn add @clerk/clerk-react @supabase/supabase-js
+      ```
+
+      ```bash filename="terminal"
+      pnpm add @clerk/clerk-react @supabase/supabase-js
+      ```
+    </CodeBlockTabs>
+  </Tab>
+</Tabs>
+
+Then, set up your environment variables:
+
+1. If you don't have a `.env.local` file in the root directory of your Next.js project, create one now.
+1. Find your Clerk publishable key and secret key. If you're signed into Clerk, the `.env.local` snippet below will contain your keys. Otherwise:
+    - Navigate to your Clerk Dashboard.
+    - Select your application, then select **API Keys** in the sidebar menu.
+    - You can copy your keys from the Quick Copy section.
+1. Add your keys to your `.env.local` file.
+1. Find your Supabase credentials:
+    - Go to your Supabase dashboard. In the sidebar, select **Settings** > **API**.
+    - Copy the **Project URL**.
+    - Copy the value beside `anon` `public` in the **Project API Keys** section.
+1. Add your Supabase credentials to your `.env.local` file.
+
+The final result should be similar to this:
+
+    <InjectKeys>
+    <CodeBlockTabs type="framework" options={["Next.js", "React"]}>
+    ```js filename=".env.local"
+    NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+    NEXT_PUBLIC_SUPABASE_URL=your-supabase-url
+    NEXT_PUBLIC_SUPABASE_KEY=your-supabase-anon-key
+    ```
+    ```js filename=".env.local"
+    REACT_APP_CLERK_PUBLISHABLE_KEY={{pub_key}}
+    REACT_APP_SUPABASE_URL=your-supabase-url
+    REACT_APP_SUPABASE_KEY=your-supabase-anon-key
```

Comment:
the pub_key will show up as `your_publishable_key` so let's use `_` to be consistent with naming conventions !

```suggestion
    NEXT_PUBLIC_SUPABASE_URL=your_supabase_url
    NEXT_PUBLIC_SUPABASE_KEY=your_supabase_anon_key
    ```
    ```js filename=".env.local"
    REACT_APP_CLERK_PUBLISHABLE_KEY={{pub_key}}
    REACT_APP_SUPABASE_URL=your_supabase_url
    REACT_APP_SUPABASE_KEY=your_supabase_anon_key
```